lint-install.go

```diff
@@ -258,7 +258,7 @@ func goLintCmd(root string, level string, fix bool) string {
 		return fmt.Sprintf("$(GOLANGCI_LINT_BIN) run%s", suffix)
 	}
 
-	return fmt.Sprintf(`find . -name go.mod -execdir "$(GOLANGCI_LINT_BIN)" run -c "$(GOLINT_CONFIG)"%s \;`, suffix)
+	return fmt.Sprintf(`find . -name go.mod -execdir "$(CURDIR)/$(GOLANGCI_LINT_BIN)" run -c "$(GOLINT_CONFIG)"%s \;`, suffix)
```

Comment:
We already have [LINT_ROOT](https://github.com/tinkerbell/lint-install/blob/main/Makefile.tmpl#L11) which  I think we should use instead of CURDIR, to avoid having 2 variables with similar values. Also, I think it would be better to update the TOOL_BIN variable(s) in Makefile.tmpl instead of fixing here. That way the Makefiles are where most of the logic lives and the go binary is mostly just dealing with enabling certain functionality. We wouldn't be mixing logic among 2 different prcoesses/times ("buildtime" and "runtime"). We should probably just do this for all the binaries.

Reply:
@tstromberg thoughts on this?